ert_gui/tools/plot/plot_window.py

```diff
@@ -186,13 +184,23 @@ def keySelected(self):
             for widget in self._plot_widgets
             if widget._plotter.dimensionality == key_def["dimensionality"]
         ]
-        self._central_tab.currentChanged.disconnect()
+
+        current_widget = self._central_tab.currentWidget()
         for plot_widget in self._plot_widgets:
             self._central_tab.setTabEnabled(
                 self._central_tab.indexOf(plot_widget), plot_widget in available_widgets
             )
-        self._central_tab.currentChanged.connect(self.currentPlotChanged)
```

Comment:
Was this not needed?

Reply:
It's set here:

https://github.com/equinor/ert/blob/3d7ddf7cb832b915b817b6b45af14c7bbf9b9f72/ert_gui/tools/plot/plot_window.py#L64

Not sure why it is set again on line 194, but it's not needed as far as I can tell.
Am I missing something?

Reply:
Not sure, but was disconnected on 189, then added again here

Reply:
To me it seems unnecessary but I might well be wrong.
Could you help us out here @xjules ?

Reply:
> To me it seems unnecessary but I might well be wrong. Could you help us out here @xjules ?

Looking at it now the slot does connect in the constructor; so it might have been another reason. Let me find the corresponding issue.

Reply:
It does seem a bit strange to disconnet it and then immediately reconnecting, but will wait for @xjules. Other than that the PR looks good 👍

Reply:
> > To me it seems unnecessary but I might well be wrong. Could you help us out here @xjules ?
> 
> Looking at it now the slot does connect in the constructor; so it might have been another reason. Let me find the corresponding issue.

It was the following issue: https://github.com/equinor/ert/issues/1282
You can try to remove it, but make sure to test that the behaviour is as expected in 🔝 @dafeda